### DISCUSSION TITLE
Support numeric range query

### DISCUSSION THREAD

**Discussion by arsaboo** [MoeMusic/Moe#193] [Ideas]
It would be great if we can use numerical values in queries. For example, year greater than 2010 or year between 1990 and 2000. This does not have to be limited to `year`, but can be extended to other fields.

**Comment by jtpavlock** [comment]
Great idea, would you mind opening a feature request issue for it?

**Comment by arsaboo** [comment]
Done https://github.com/MoeMusic/Moe/issues/199
